testing/scripts/playwright_test.py

```
@@ -4694,6 +4694,222 @@ async def test_tree_view(page: Page) -> None:
+    await navigate(page, APP_URL + "?tab=ipv6")
+    await page.click("#panel-ipv6 .tool-trigger[data-tool='supernet6']")
+    await page.wait_for_selector("#panel-ipv6 .tool-drawer.open")
+    await page.fill("textarea[name='supernet6_input']", "2001:db8::/64\nfd00::/64")
+    await page.click("button[name='supernet6_action'][value='find']")
+    await page.wait_for_load_state("load")
+    sup6_panel = page.locator("#panel-ipv6 .overlap-panel").filter(
+        has=page.locator("textarea[name='supernet6_input']")
+    )
+    err_count = await sup6_panel.locator(".error").count()
+    assert_true("supernet6: disjoint roots do not raise error band", err_count == 0,
+                f"unexpected error band on disjoint roots")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)